js/index.js

```diff
@@ -1 +1,36 @@
-// Your code goes here
+let navLinks = document.querySelectorAll('a');
+
+navLinks.forEach(element => {
+    element.addEventListener("mouseover", function(event){
+        event.target.style.color = "#A0001E"
+    });
+});
+
+document.addEventListener('keydown', logKey);
+
+function logKey(e) {
+    log.textContent += ` ${e.code}`;
```

Comment:
Really good knowing that element IDs are added to the global scope as variables that we can access. In my opinion, I wouldn't use this too much in order to prevent confusion. 👍